pkg/kapp/config/default.go

```diff
@@ -612,6 +612,19 @@ changeRuleBindings:
           - anyMatcher: {matchers: *serviceAccountMatchers}
           - anyMatcher: {matchers: *rbacMatchers}
 
+# Delete serviceAccount (SA) before namespace (if serviceAccount is part of that namespace)
+# deletion of namespace before SA sometimes leads deletion of SA before 
+# the resources on which it is dependent on like kapp-controller-packageInstall
```

Comment:
```suggestion
# Delete namespaces after deleting namespaced SAs so that resources like 
# kapp-controller PackageInstalls can be deleted gracefully.
```

Reply:
Let's also move this is a bit up a bit (along with the other namespace and namespaced resource rule).

Reply:
resolved in PR https://github.com/vmware-tanzu/carvel-kapp/pull/593/files